oxd-server/src/main/resources/oxd-server.yml

```diff
@@ -24,6 +24,10 @@ enable_tracing: false
 tracer: 'jaeger'
 tracer_host: 'localhost'
 tracer_port: 5775
+proxy_configuration:
```

Comment:
No need to keep empty configuration. It's empty by default. We definitely need it in docs.

Reply:
Done